packs/feats/scholarly-defense.json

```diff
@@ -6,6 +6,24 @@
         "actionType": {
             "value": "passive"
         },
+        "rules": [
```

Comment:
This should probably be in the rules section below.

Reply:
Please add rules in Foundry and extract them instead of adding them in an editor. Even if they are added to the right array it may just end up making extract noise for us on later PRs when the extraction re-orders the properties.

Reply:
Kk